code-challenges.test.js

```diff
@@ -74,18 +185,42 @@ var nums2 = [109, 5, 9, -59, 8, 24]
 // --------------------4) Create a function that takes in a string and returns a string with every other letter capitalized.
 
 // a) Create a test with expect statements for each of the variables provided.
+describe("when upperCased is called", () =>{
+    it("if argument is testString1, return aLbAtRoSs",() =>{
+        const argument = testString1;
+        const actualResults = upperCased(argument)
+        expect(actualResults).toEqual("aLbAtRoSs")
+    })
+    it("if argument is testString2, return jAbBeRwOcKy",() =>{
+        const argument = testString2;
+        const actualResults = upperCased(argument)
+        expect(actualResults).toEqual("jAbBeRwOcKy")
+    })
+})
+
+// b) Create the function that makes the test pass.
+
+// create a function that takes a string as an argument and returns every other letter caputalized
+const upperCased = (array) => {
+    let upCassedArray = []
+    for (i= 0 ; i< array.length; i++){
+        upCassedArray += i %2 !== 0 ? array.charAt(i).toUpperCase() : array.charAt(i);
```

Comment:
just a formatting suggestion when using long statements involving ternary operators
```suggestion
    for (i= 0 ; i< array.length; i++){
        upCassedArray += i %2 !== 0 
          ? array.charAt(i).toUpperCase() 
          : array.charAt(i);
```